docs/site/Contributions.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I thought we've discussed this will not be included because we don't have anything to put it there?

Reply:
It was supposed to be removed. Must have been a git error again. I will remove it (again)!